frontend/public/components/storage/storage.js

```diff
@@ -0,0 +1,63 @@
+import * as _ from "lodash-es";
```

Comment:
Prefer TypeScript for new files.

This doesn't look like there are any component, just a collection of utilities. It seems like this is the wrong folder for this file.

I might just put this with the form if it's not used anywhere else.

Reply:
Sam, in an attempt to keep the files more readable and shorter, I like to break some of the pure JS stuff out into a separate file.  Can I just move that file to a different location that is more appropriate?  Or do you feel strongly about removing it and combing it with the form? I am good with either.

Reply:
General k8s utilities probably should go under `modules/k8s/`, but these seem so specific to the form, I'd probably keep them with the form.

Reply:
I removed the file.